python/cuml/test/dask/test_input_utils.py

```diff
@@ -35,15 +41,23 @@ def test_extract_partitions_worker_list(nrows, ncols, n_parts, input_type,
 @pytest.mark.parametrize("nrows", [24])
 @pytest.mark.parametrize("ncols", [2])
 @pytest.mark.parametrize("n_parts", [2, 23])
-@pytest.mark.parametrize("input_type", ["dataframe", "array"])
+@pytest.mark.parametrize("input_type", ["dataframe", "array", "series"])
 @pytest.mark.parametrize("colocated", [True, False])
 def test_extract_partitions_shape(nrows, ncols, n_parts, input_type,
                                   colocated, cluster):
     client = Client(cluster)
 
     try:
-        X, y = make_blobs(nrows=nrows, ncols=ncols, n_parts=n_parts,
-                          output=input_type)
+        if input_type == "series":
```

Comment:
I think you could clean this up a little bit if you used some normalized `adjusted_input_type` that is separate from `input_type` argument. 

```suggestion
        adj_input_type = "dataframe" if input_type == "series" else input_type
        
        X, y = make_blobs(nrows=nrows, ncols=ncols, n_parts=n_parts,
                              output=adj_input_type)
        if input_type == "series":
            X = X[X.columns[0]]
            y = y[y.columns[0]]
        elif input_type == "dataframe":
            ...
                 
```

Reply:
Should be able to do something similar in the test above as well.

Reply:
Good point! I was myself debating the best way to make it more readable but I think your solution definitely is clearer.